css/style.css

```diff
@@ -0,0 +1,28 @@
+.date {
+  position: relative;
+  font-family: "Roboto";
```

Comment:
Let's add some fallback font. Usually, we set `font-family` for all the pages instead of a single class, unless specified otherwise.

Reply:
I removed it because it was not required